### ISSUE TITLE
[BUG]: Colour Accessibility

### ISSUE THREAD

**Issue by RachBreeze** [umbraco/Umbraco.UI#601]
Hello

**Describe the bug**
The UUI reports the colors used in the back office are inaccessible, this is something we've discused a few times, but hace had concern re branding. With the new back office it would be great if we could move to more accessible colours. 

**To Reproduce**
This is reported in the UUI https://uui.umbraco.com/?path=/story/uui-button-group--mixed-looks-and-colors

**Expected behavior**
The colours to comply with WCAG 

**Additional context**
Apologies if this is the wrong place to raise the issue.

**Comment by iOvergaard** [comment]
Hi @RachBreeze 
Thanks for the report. I have taken the liberty to move it to the Umbraco.UI repository, since it looks like something that should be fixed there #h5yr
